### PR TITLE
Added `ZydisEncoderNopFill`

### PR DESCRIPTION
**Pull request by mappzor**
Added small helper function to fill buffer with nops. Useful for padding, "erasing" code, etc.

### CODE REVIEW

**Review by athre0z — APPROVED** [review]
LGTM, thanks!

File: src/Encoder.c [hunk 1]
```diff
@@ -4706,4 +4706,30 @@ ZYDIS_EXPORT ZyanStatus ZydisEncoderDecodedInstructionToEncoderRequest(
     return ZYAN_STATUS_SUCCESS;
 }
 
+ZYDIS_EXPORT void ZydisEncoderNopFill(void *buffer, ZyanUSize length)
+{
```
Comment (flobernd) [inline src/Encoder.c:4710]: We should add the "invalid argument" checks here for consistency (`buffer == null`, `length < 0`).

Comment (athre0z) [reply src/Encoder.c:4710]: Length == 0 is a legitimate input for this function IMO. It should just do nothing in this case.

Comment (flobernd) [reply src/Encoder.c:4710]: Yes, but `length > 0` should error imho.

Comment (athre0z) [reply src/Encoder.c:4710]: This case is already eliminated by the argument being unsigned, unless I’m missing something.

Comment (flobernd) [reply src/Encoder.c:4710]: Ah yes, true. So just the buffer I guess :p

**Pull request merged**